core/src/test/scala/unit/kafka/server/KafkaMetricReporterExceptionHandlingTest.scala

```diff
@@ -50,19 +52,30 @@ class KafkaMetricReporterExceptionHandlingTest extends BaseRequestTest {
     // need a quota prop to register a "throttle-time" metrics after server startup
     val quotaProps = new Properties()
     quotaProps.put(QuotaConfigs.REQUEST_PERCENTAGE_OVERRIDE_CONFIG, "0.1")
-    adminZkClient.changeClientIdConfig("<default>", quotaProps)
+
+    if (isKRaftTest()) {
+      val admin = createAdminClient()
```

Comment:
Can we make this part into `QuoramTestHarness`, just like the ZK client?

Reply:
That's good idea. Reusing this part seems much better.